src/App.js

```diff
@@ -1,23 +1,53 @@
-import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json';
+import { useState } from 'react';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import SearchBar from './components/SearchBar';
 
 function App() {
+  const [allFood, setAllFood] = useState(foods);
+  const [showFood, setShowFood] = useState(foods);
+
+  const createFood = (food) => {
+    const newFood = [food, ...foods];
+    setAllFood(newFood);
+    setShowFood(newFood);
+  };
+
+  const deleteFood = (name) => {
+    const toDeleteFood = foods.filter((food) => food.name !== name);
```

Comment:
Here you should pass `allFood.filter((food)` instead of `foods.filter((food)` so your delete button will work just fine